capstone/src/main/webapp/youtubeScript.js

```diff
@@ -21,14 +21,34 @@ const ATTRIBUTES_BY_LANGUAGE = {
   'pt': ['TOXICITY', 'SEVERE_TOXICITY', 'IDENTITY_ATTACK', 'INSULT', 'PROFANITY', 'THREAT']
 };
 
+const YOUTUBE_CATEGORIES = {
+  'Autos&Vehicles': 2,
+  'Comedy': 23,
+  'Entertainment': 24,
+  'Film&Animation': 1,
+  'Gaming': 20,
+  'How-to&Style' : 26,
+  'Music': 10,
+  'Pets&Animals': 15,
+  'Science&Technology': 28,
+  'Sports' : 17,
+};
+
 /** Calls youtube servlet and passes output to perspctive */
 async function callYoutube() {
+  document.getElementById('search-type').innerHTML = ""
```

Comment:
nit: missing a terminating ";"

Reply:
addressed in "spacing and address Daniel's nits"